sway/input/cursor.c

```diff
@@ -984,6 +984,26 @@ static void handle_pointer_swipe_end(struct wl_listener *listener, void *data) {
 			event->time_msec, event->cancelled);
 }
 
+static void handle_pointer_hold_begin(struct wl_listener *listener, void *data) {
+	struct sway_cursor *cursor = wl_container_of(
+			listener, cursor, swipe_end);
```

Comment:
Ah, one other thing: this should read `hold_begin`, not `swipe_end`. Same below.

Reply:
Whoops, thought I got these, thanks!